### PR TITLE
bug(refs T30938): add permission for paragraphs in statements

### PR DESCRIPTION
**Pull request by gruenbergerdemos**
**Ticket:** https://yaits.demos-deutschland.de/T30938

There aren't paragraphs and location in statements in every project, so we need a permission for it.

### How to review/test
Login as citizen, go to a procedure and write a statement. There should be no option for paragraphs or location. 

### Linked PRs (optional)
https://github.com/demos-europe/demosplan-project-teilhabe/pull/10
https://github.com/demos-europe/demosplan-project-robobsh/pull/15
https://github.com/demos-europe/demosplan-project-regio/pull/18
https://github.com/demos-europe/demosplan-project-planfestsh/pull/15
https://github.com/demos-europe/demosplan-project-diplanbau/pull/53
https://github.com/demos-europe/demosplan-project-bobsh/pull/16
https://github.com/demos-europe/demosplan-project-bobhh/pull/21
https://github.com/demos-europe/demosplan-project-bimschgsh/pull/19
https://github.com/demos-europe/demosplan-project-ewm/pull/27
https://github.com/demos-europe/demosplan-project-blp/pull/27

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Resources/config/permissions.yml
Comment (martendemos) [inline demosplan/DemosPlanCoreBundle/Resources/config/permissions.yml:2587]: We should be very sure that there is not already a permission for that. 
In many other cases the option to link a paragraph is correctly hidden for this project, that's why I guess there is already a permission which we should use.

**Review by martendemos — CHANGES_REQUESTED** [review]
I'm looking for a existing permission we can use, to avoid creating an additional permission, which functionality is already covered.

File: demosplan/DemosPlanCoreBundle/Resources/config/permissions.yml
Comment (gruenbergerdemos) [reply demosplan/DemosPlanCoreBundle/Resources/config/permissions.yml:2587]: Maybe `field_procedure_paragraphs`?

Comment (martendemos) [reply demosplan/DemosPlanCoreBundle/Resources/config/permissions.yml:2587]: for refs:
`feature_documents_new_statement` it is

Comment (gruenbergerdemos) [reply demosplan/DemosPlanCoreBundle/Resources/config/permissions.yml]: Changed in [cf93172](https://github.com/demos-europe/demosplan-core/pull/628/commits/cf931724ee7820b5a87f4e6e74e549c0e2324f80)

**Review by martendemos — DISMISSED** [review]

**Comment by martendemos** [comment]
Do not merge this PR not until after the other listed PRs are merged.

**Pull request merged**
